de/cheatsheet-deep-learning.md

```diff
@@ -144,13 +144,13 @@
 
 **25. Types of gates ― Here are the different types of gates that we encounter in a typical recurrent neural network:**
 
-&#10230; Typen von Gattern - Dies sind die verschiedenen Typen der Gatter die wir in typischen rekurrenten neuronalen Netzen vorfinden:
+&#10230; Typen von Toren - Dies sind die verschiedenen Typen der Tore die wir in typischen rekurrenten neuronalen Netzen vorfinden:
 
 <br>
 
 **26. [Input gate, forget gate, gate, output gate]**
 
-&#10230; [Eingangsgatter, Vergißgatter, Gatter, Ausgangsgatter]
+&#10230; [Eingangstor, Vergesstor, Gatter, Ausgangstor]
```

Comment:
[Input-Gatter, Forget-Gatter, Output-Gatter, Speicher-Gatter]

Reply:
Have stuck to German words ("Eingangsgatter" instead of "Input-Gatter"). Acceptable? Also, is "Gate" "Speichergatter"?